mis_builder/models/mis_builder.py

```diff
@@ -668,7 +668,9 @@ def _compute(self, cr, uid, lang_id, c, aep, context=None):
                     kpi_val_rendered = kpi_obj.render(
                         cr, uid, lang_id, kpi, kpi_val, context=context)
 
-                localdict[kpi.name] = kpi_val
+                if kpi_val is not None:
+                    localdict[kpi.name] = kpi_val
```

Comment:
It makes sense, indeed. I wish I'd understand why it works on 8.0.

Instead of this test, I'd move  `localdict[kpi.name] = kpi_val` before the `except ZeroDivisionError`